python/ray/serve/tests/test_deployment_state.py

```diff
@@ -1788,9 +1981,178 @@ def test_new_version_and_scale_up(mock_deployment_state):
 
     deployment_state.update()
     assert deployment_state.curr_status_info.status == DeploymentStatus.HEALTHY
+    assert (
+        deployment_state.curr_status_info.status_trigger
+        == DeploymentStatusTrigger.CONFIG_UPDATE
+    )
+
+
+@pytest.mark.parametrize("scale_direction", ["up", "down"])
+def test_scale_num_replicas_up(mock_deployment_state_manager_full, scale_direction):
```

Comment:
Maybe rename the test to drop the suffix "_up" 🙃 same goes to "test_autoscale_up"